.github/workflows/samples_configuration.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         uses: AliSajid/random-wait-action@main
         with:
           minimum: 1
-          maximum: 300
```

Comment:
The range of this plugin has been changed to 1-99 otherwise the workflow will fail.